NebulaClient/PacketProcessors/Session/PlayerJoiningProcessor.cs

```diff
@@ -12,7 +13,7 @@ public class PlayerJoiningProcessor : IPacketProcessor<PlayerJoining>
         public void ProcessPacket(PlayerJoining packet, NebulaConnection conn)
         {
             SimulatedWorld.SpawnRemotePlayerModel(packet.PlayerData);
-            GameMain.Pause();
+            typeof(GameMain).GetField("_paused", BindingFlags.Instance | BindingFlags.NonPublic).SetValue(GameMain.instance, true);
```

Comment:
Hmmm, is Harmony referenced? This could use AccessTools, which would set this in an optimized manner.

Reply:
https://harmony.pardeike.net/api/HarmonyLib.AccessTools.html#HarmonyLib_AccessTools_FieldRefAccess__2___0_System_Reflection_FieldInfo_